src/bar.jl

```diff
@@ -4,10 +4,10 @@ recipetype(::Val{:groupedbar}, args...) = GroupedBar(args)
 
 Plots.group_as_matrix(g::GroupedBar) = true
 
-grouped_xy(x::AbstractVector, y::AbstractMatrix) = x, y
-grouped_xy(y::AbstractMatrix) = 1:size(y,1), y
+grouped_xy(x::AbstractVector, y::AbstractArray) = x, y
+grouped_xy(y::AbstractArray) = 1:size(y,1), y
 
-@recipe function f(g::GroupedBar)
+@recipe function f(g::GroupedBar; spacing = 0.1)
```

Comment:
Like the name `spacing`! Somehow I feel it should default to `0` to preserve the old behavior of `groupedbar`. I wonder if there is a way to make it clear that we refer to the spacing "within groups" whereas the ration "spacing between groups / spacing within groups" IIUC is hardcoded.

Reply:
I've changed the default spacing of `groupedbar` to `0` again. Is it ok toleave it at `0.1` for `groupedboxplot` and `groupedviolin`? The spacing between groups can be set with the `bar_width` attribute.